lib/mongoose_push/logger/fmt.ex

```diff
@@ -0,0 +1,39 @@
+defmodule MongoosePush.Logger.Fmt do
+  @moduledoc """
+  Module responsible for formatting logger output
```

Comment:
This moduledoc could be updated to reflect that is only doing a specific kind of formatting now 🙂